app/src/editor/canvas/components/Resizable/resizeable.stories.jsx

```diff
@@ -24,6 +24,8 @@ stories.add('default', () => (
             <Resizeable
```

Comment:
unrelated but just noticed this:  `Resizeable` (incorrect) vs `Resizable` (correct) in `resizeable.stories.jsx` and in the name here.

Reply:
Fixed in `development`.